lib/contracts/portal/deploy.go

```diff
@@ -36,6 +36,10 @@ const (
 	XReceiptMaxErrorBytes = uint16(256)
 )
 
+func isDeadOrEmpty(addr common.Address) bool {
+	return addr == common.Address{} || addr == common.HexToAddress(eoa.ZeroXDead)
+}
+
```

Comment:
Saw we use `0x000...dead` in `e2e/app/eoa` so added these checks in a few places to make sure we don't set owner to dead or something like that